Keycloak 2.5.5

jsight opened the pull request.

mareknovotny commented:
I guess this PR is mixed with https://github.com/windup/windup-web/pull/524 and it failed tests.

jsight commented:
@mareknovotny - I have removed the extranneous commit and also fixed the web config to actually work with 2.5.5 (the configuration has changed quite a bit since the previous versions of Keycloak). The tests should pass now.

mareknovotny commented:
retest
